templates/team.html

```diff
@@ -43,4 +72,57 @@ <h4 class="label">{{.Specialization}}</h4>
 		{{end}}
 	</section>
 </main>
+
+<script src="/static/js/filter-core.js"></script>   <!-- shared by research page -->
+<script>
+(() => {
+  const cards   = document.querySelectorAll('.team-member');
```

Comment:
I won't be reviewing the code below but I tested it works and it looks fine at a glance!